PackageInfo.g

```diff
@@ -119,8 +111,38 @@ Autoload := false,
 Keywords := [
   "Knuth-Bendix",
   "Automatic Groups"
-]
+],
 
 #TestFile := "tst/testall.g",
 
+AutoDoc := rec(
+    TitlePage := rec(
+        Copyright := Concatenation(
+            "&copyright; 1997 by Derek Holt<P/>\n\n",
+            "This package may be distributed under the terms and conditions of the\n",
+            "GNU Public License Version 2.\n"
+            ),
+        Abstract := Concatenation( 
+            "The &KBMAG; package is a &GAP; interface to some `C' ",
+            "programs for running the Knuth-Bendix completion program ", 
+            "on finite semigroup, monoid or group presentations, ", 
+            "and for attempting to compute  automatic structures ", 
+            "of finitely presented groups.<P/>", 
+            "Bug reports, comments, suggestions for additional features, and ", 
+            "offers to implement some of these, will all be very welcome.<P/>", 
+            "Please submit any issues at ", 
+            "<File>http://github.com/gap-packages/kbmag/issues/</File>.<P/>" 
+            ), 
+        Acknowledgements := Concatenation( 
+            "This documentation was prepared with the ", 
+            "<Package>GAPDoc</Package> package <Cite Key='GAPDoc' /> ", 
+            "of Frank L&#xfc;beck and Max Neunh&#xf6;ffer.<P/>", 
```

Comment:
Just write the umlauts as unicode, i.e. "Frank Lübeck and Max Neunhöffer" -- we already use UTF-8 in that file, e.g. for my address in "Gießen"